test/polly.jl

```diff
@@ -0,0 +1,156 @@
+# This file is a part of Julia. License is MIT: http://julialang.org/license
```

Comment:
you'll need to add this file to the list in `test/choosetests.jl` if you want it to run